study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/PdfBuilder.java

```diff
@@ -268,23 +270,33 @@ private PdfTemplate buildPdfTemplate(Handle handle, Config fileCfg) {
             throw new DDPException(e);
         }
 
+        Long languageCodeId = LanguageStore.getDefault().getId();
```

Comment:
Will it be better if we try figuring out which language the study has set as the "default" and using that instead?